src/index.html

```diff
@@ -22,6 +22,412 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="https://www.moyo.ua/ua/"
+          class="header__logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="menu">
+          <a
+            href="#"
+            class="menu__link is-active"
+          >
+            Apple
+          </a>
+          <a
+            href="#"
+            class="menu__link"
+          >
+            Samsung
+          </a>
+          <a
+            href="#"
+            class="menu__link"
+          >
+            Smartphones
+          </a>
+          <a
+            href="#"
+            class="menu__link"
+            data-qa="nav-hover"
+          >
+            Laptops & Computers
+          </a>
+          <a
+            href="#"
+            class="menu__link"
+          >
+            Gadgets
+          </a>
+          <a
+            href="#"
+            class="menu__link"
+          >
+            Tablets
+          </a>
+          <a
+            href="#"
+            class="menu__link"
+          >
+            Photo
+          </a>
+          <a
+            href="#"
+            class="menu__link"
+          >
+            Video
+          </a>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="main__container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__wrapper">
+            <a
+              href="#"
+              class="card__link-img"
+            >
+              <img
+                src="images/imac.jpeg"
+                alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA)"
+                class="card__img"
+              />
+            </a>
+            <a
+              href="#"
+              class="card__link-title"
+            >
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </a>
+            <p class="card__product-code">Product code: 195434</p>
+            <div class="card__review">
+              <ul class="card__stars stars">
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+              </ul>
+              <span class="card__review-text">Reviews: 5</span>
+            </div>
+            <div class="card__price price">
+              <p class="price__title">Price:</p>
+              <p class="price__value">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="card__button button"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+        <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
        </div>
        
        <div class="card">
```